src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -61,7 +62,7 @@ export default class QuestionnaireForm extends Component {
 
   componentWillMount() {
     // search for any partially completed QuestionnaireResponses
-    if (this.props.standalone) {
+    if (this.props.response) {
```

Comment:
Is standalone launch totally replaced by relaunch?

Reply:
no but this conditional and ensuing code is functional for both standalone launch and regular launch with just a response in the props, so instead of just checking for standalone this section of code is now multipurpose.